build.sbt

```diff
@@ -122,6 +123,11 @@ lazy val httpMarshallersJava = project("akka-http-marshallers-java")
 lazy val httpJackson =
   httpMarshallersJavaSubproject("jackson")
 
+lazy val httpCaching = project("akka-http-caching")
```

Comment:
The decision to put it all under akka-http-caching was based on @ktoso 's [original comment](https://github.com/akka/akka-http/issues/213#issuecomment-271832937). I'm open to changing this though